drivers/media/i2c/ov5693.c

```diff
@@ -1678,7 +1678,7 @@ static int ov5693_configure_gpios(struct ov5693_device *ov5693)
         }
 
         ov5693->indicator_led = gpiod_get_optional(&ov5693->client->dev, "indicator-led",
-                                        GPIOD_OUT_HIGH);
+                                        GPIOD_OUT_LOW);
```

Comment:
My LED still flashes on startup but that's a minor issue.

Reply:
That's...weird. I'd love to understand why it still does that...maybe the polarity is different between the machines, then...but then that would mean the LED outputs when the sensor turns on / off would be reversed.

Pass; but like you say pretty minor. I'll think about that one in the future.